doc/tutorials/using_ubuntu_as_sos.rst

```diff
@@ -0,0 +1,262 @@
+in Ubuntu. We will therefore download the source code, build it, and install it locally.
+
+1. Install the build tools and dependencies
+
+   Follow the instructions found in the :ref:`getting_started` to install all the build tools
+   and dependencies on your system.
+
+#. Clone the `Project ACRN <https://github.com/projectacrn/acrn-hypervisor>`_ code repository
+
+   .. code-block:: console
+
+      cd ~
+      git clone https://github.com/projectacrn/acrn-hypervisor
+      git checkout <known-good-tag/release>
+
+   .. note::
+      We clone the git repository below but it is also possible to download the tarball for any specific
+      tag or release from the `Project ACRN Github release page <https://github.com/projectacrn/acrn-hypervisor/releases>`_
+
+#. Build and install ACRN
+
+   Here is the short version of how to build and install ACRN from source.
+
+   .. code-block:: console
+
+      cd ~/acrn-hypervisor
+      make PLATFORM=uefi
+      sudo make install
+
+   For more details, please refer to the :ref:`getting_started`.
+
+#. Install the hypervisor
+
+   The ACRN devicemodel and tools were installed as part of the previous step. However, ``make install``
+   does not install the hypervisor (``acrn.efi``) on your EFI System Partition (ESP), nor does it
+   configure your EFI firmware to boot it automatically. Follow the steps below to perform these
+   operations and complete the ACRN installation.
+
+   #. Mount the EFI System Partition (ESP) and add the ACRN hypervisor and Service OS kernel to it (as ``root``)
+
+      .. code-block:: console
+
+         sudo umount /boot/efi
+         sudo lsblk
+         sudo mount /dev/sda1 /mnt
+         ls /mnt/EFI/ubuntu
+
+      You should see the following output:
+
+      .. code-block:: none
+
+         fw  fwupx64.efi  grub.cfg  grubx64.efi  MokManager.efi  shimx64.efi
+
+   #. Install the hypervisor (``acrn.efi``)
+
+      .. code-block:: console
+
+         sudo mkdir /mnt/EFI/acrn/
+         sudo cp ~/acrn-hypervisor/build/hypervisor/acrn.efi /mnt/EFI/acrn
+
+   #. Configure the EFI firmware to boot the ACRN hypervisor by default
+
+      .. code-block:: console
+
+         sudo efibootmgr -c -l "\EFI\acrn\acrn.efi" -d /dev/sda -p 1 -L "ACRN Hypervisor" -u "bootloader=\EFI\ubuntu\grubx64.efi"
+         sudo efibootmgr -v # to verify that the "ACRN Hypervisor" will be booted first
+
+      You can change the boot order at any time using ``efibootmgr -o XXX,XXX,XXX``
+
+      .. note::
+         By default, the “ACRN Hypervisor” you have just added should be the first one to boot but it is a recommended
+         that you double-check that using ``efibootmgr -v`` or by entering the EFI firmware (``F10``)
+
+Install the Service OS kernel
+*****************************
+
+You can download latest Service OS kernel from `<https://download.clearlinux.org/releases/current/clear/x86_64/os/Packages/>`_
```

Comment:
Need to have a unique URL here instead of `releases/current` which is a moving target